commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Activity.kt

```diff
@@ -548,6 +548,15 @@ fun BaseSimpleActivity.launchCallIntent(recipient: String, handle: PhoneAccountH
                 putExtra(TelecomManager.EXTRA_PHONE_ACCOUNT_HANDLE, handle)
             }
 
+            if (isDefaultDialer()) {
+                val packageName = when(BuildConfig.DEBUG) {
```

Comment:
lets use an if/else...

Reply:
You mean instead of `when` use `if-else`?

Reply:
ye, "when" with "true" and "false" is just so unprefessional and looks wrong